### ISSUE TITLE
Dequeuing cells in Swift

### ISSUE THREAD

**Issue by hartbit** [johnno1962/InjectionApp#12]
In Swift, when de-queuing custom cells who's class has been swizzled causes a crash:

```
Could not cast value of type 'MyApp.SectionHeaderView' (0x10e9afee0) to 'MyApp.SectionHeaderView' (0x123532bb8).
```

**Comment by johnno1962** [comment]
Intersesting.. Any chance you would be able to put together a small example project and I’ll take a look.

**Comment by hartbit** [comment]
I’ll try this evening.

**Comment by johnno1962** [comment]
Cheers, I have an idea what it is but not quite what to do about it. Swizzled instance has different “isa"

**Comment by hartbit** [comment]
Yeah, was thinking the same, and it doesn't work with a Swift runtime cast.

**Comment by johnno1962** [comment]
I found I could only replicate this when the cell class was in the same file as the UITableViewController subclass. Could this be the solution?

**Comment by hartbit** [comment]
I'm in that case!
